mp-istio-sample/README.md

```diff
@@ -0,0 +1,115 @@
+## MicroProfile Istio Sample
+This project is intended to show the interaction between Istio and the various MicroProfile specifications.
+
+### Requirements
```

Comment:
I think it would be beneficial to separate out what's a general sample requirement/setup step vs anything required for a particular implementation, such as Liberty.

Having separate doc files might be better as it's a clearer separation